src/components/forms/slider.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import bindAll from 'lodash.bindall';
+
+import styles from './slider.css';
+
+const CONTAINER_WIDTH = 150;
+const HANDLE_WIDTH = 26;
+
+class SliderComponent extends React.Component {
+    constructor (props) {
+        super(props);
+        bindAll(this, [
+            'handleMouseDown',
+            'handleMouseUp',
+            'handleMouseMove',
+            'setBackground'
+        ]);
+    }
+
+    handleMouseDown () {
+        document.addEventListener('mouseup', this.handleMouseUp);
+        document.addEventListener('mousemove', this.handleMouseMove);
+    }
+
+    handleMouseUp () {
```

Comment:
Is it possible that the slider will catch the mouse down but not the mouse up and accumulate event listeners?